sandbox/src/services/patient-searcher.js

```diff
@@ -64,11 +64,17 @@ function buildPatientResponse(examplePatients = [], searchScore = 1.0) {
                 resource: patient,
             })
         });
+        return response;
     } else {
-        delete response.entry
-    }
+        return {
```

Comment:
Do we have any coding standards for indentation of nested dictionaries?

I can't see anything from a quick look but I think this style is hiding a lot of detail which can be shown with a different style

`{
    resourceType: "OperationOutcome",
    timestamp: datefns.format(Date.now(), "yyyy-MM-dd'T'HH:mm:ss+00:00"),
    "issue": [
        {
            "severity": "warning",
            "code": "not-supported", 
            "details": {
                "coding": [
                    {
                        "system": "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode", 
                        "version": "1",
                        "code": "not-supported",
                        "display": "Unsupported operation"
                    }
                ]
            }, 
            "diagnostics": "This mock endpoint has no example response for this combination of search parameters"
        }
    ]
}`

(I'm not suggesting that style is the one that should be used)

Reply:
That's using the established template for an error message, there is no scope to change it in this ticket - but certainly it's something that could be reviewed/raised under tech debt if needed.

Reply:
converted to boom.notImplemented error.